src/Github.psm1

```
@@ -1,5 +1,189 @@
+    .PARAMETER Variables
+        Hashtable of variables for the GraphQL query.
+    .PARAMETER UseFallback
+        If set, falls back to REST API on GraphQL failure.
+    .PARAMETER MaxRetries
+        Maximum number of retry attempts on rate limit. Default is 3.
+    .EXAMPLE
+        Invoke-GithubGraphQL -Query 'query { viewer { login } }'
+    #>
+    param(
+        [Parameter(Mandatory, ValueFromPipeline)]
+        [String] $Query,
```

PSScriptAnalyzer (1.24.0)

[warning] Command accepts pipeline input but has not defined a process block.

(PSUseProcessBlockForPipelineCommand)